src/main/java/org/jenkinsci/plugins/sonargerrit/inspection/entity/Issue.java

```diff
@@ -94,10 +96,50 @@ public boolean isNew() {
         return isNew != null && isNew;
     }
 
+    public void setKey(String key) {
+        this.key = key;
+    }
+
+    public void setComponent(String component) {
+        this.component = component;
+    }
+
+    public void setLine(Integer line) {
+        this.line = line;
+    }
+
+    public void setMessage(String message) {
+        this.message = message;
+    }
+
+    public void setSeverity(Severity severity) {
+        this.severity = severity;
+    }
+
+    public void setRule(String rule) {
+        this.rule = rule;
+    }
+
+    public void setStatus(String status) {
+        this.status = status;
+    }
+
+    public void setNew(Boolean aNew) {
+        this.isNew = aNew;
+    }
+
+    public Boolean getNew() {
```

Comment:
is that correct? what about `isNew()` do we actually want this method to return null?

Reply:
it's correct (see https://stackoverflow.com/questions/39307379/java-bean-specifications-for-boolean-field), but with `is` it's more readable. -> changed